### ISSUE TITLE
Feedback on Tutorial: SAP HCP predictive services, Import a predictive demo dataset in your SAP HANA

### ISSUE THREAD

**Issue by lucamanassero** [SAPDocuments/Tutorials#937]
Hi,

my name is Luca Manassero, Head of Technology at Shamat, a startup participating to the SPA HANA Startup Challenge with SAP Germany and Cap Gemini.
In this tutorial everything seems to work fine until I try to import the values in the file. At that point, I get a long list of error messages stating:

Error: (dberror) 303 - invalid DATE, TIME or TIMESTAMP value: Error while parsing 03/01/2001 00:00:00 as DATE
15:41:07 CEST (SQL Editor) Could not execute 'INSERT into "DEMO"."TIME_SERIES" values (	'04/01/2001 00:00:00' ...'

Please note that I'm running the Web IDE Developer Workbench on a Safari browser (version 9.1.3) on Mac OS X 10.11.6 in Italy.

Any hint?

Thank you,

Luca

**Comment by lucamanassero** [comment]
Hi,
I actually found the issue: the sample data provided are in the wrong format. Date and Timestamp must be provided (as in the documentation) in the following format:
'YYYY/MM/DD HH:MM:SS'
i.e.
'2001/01/04' 00:00:00' and NOT '04/01/2001 00:00:00' as provided in your example.
Any editor with column blocks will help putting Months and Days at the right place, then data will be correctly loaded.

**Comment by adadouche** [comment]
Hi Luca,

My name is Abdel and the initial author of this tutorial. 

First of all thanks for taking the time to provide your feedback.

When building the script initially, I was able to run the attached SQL on my trial account, but also on a Factory account.

But I didn't realize that actually the HANA Workbench was using my local regional settings (I'm based out of France therefore my date format is DD/MM/YYYY).

HANA uses implicit conversion on certain data types, and "string to date" is one of them, but it uses the regional settings to do so.

To fix the issue, I've used the to_date function which includes the date format pattern to be found in the string, which will ultimately fix the issue whatever the date format is used. The change was committed today and should be generally available soon.

Thanks

@bdel
